src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -31,6 +31,9 @@ interface VcsRootLocator {
 }
 
 interface BuildLocator {
+    fun withId(buildId: BuildId): BuildLocator
```

Comment:
`withId` here is needed for `sinceBuild`. It's also easier to find in this form, because it's direct mapping of REST API. I was very surprised that it wasn't there.

`withNumber` is also needed for since build, direct mapping of REST API and because TeamCity doesn't demand build number to be unique.